ocaml/testsuite/tests/typing-implicit-source-positions/commuting.ml

```diff
@@ -0,0 +1,52 @@
+(* TEST
```

Comment:
Might be good to have a test in this file showing that labels on source positions can't commute in definitions. That is, this is bad:

```ocaml
let f : a:[%src_pos] -> b:[%src_pos] -> unit = fun ~(b:[%src_pos]) ~(a:[%src_pos]) -> ()
```

Would also be good to add some tests for partial application.